packages/store/tests/action.spec.ts

```diff
@@ -262,4 +263,16 @@ describe('Action', () => {
 
     expect(callbacksCalled).toEqual(['onObjectLiteral']);
   });
+
+  it('should be executing between dispatch and complete', () => {
```

Comment:
I would like to see more tests for different cases:   

- [x] dispatch action  (synchrony)
- [ ] dispatch action with error  (synchrony)
- [ ] dispatch async action  
- [ ] dispatch async action  with error
- [ ] dispatch multiple actions  (synchrony)
- [ ] dispatch multiple actions with errors (async)
- [ ] dispatch multiple actions (async)  
- [ ] dispatch multiple actions with errors (async)

Reply:
multiple actions are not supported

> ended up allowing a single action for now, if we allow multiple actions, it will return a status on each action which can be confusing, will try to come up with a way of notifying when all are dispatched and all are completed @arturovt

Reply:
@joaqcid 

1) Why `new Action1()` instead `Action1`?

prefer 

```ts
actions.pipe(ofActionExecuting(Action1)).subscribe(executing => actionStatus.push(executing));
```

2) should return object information instead boolean

instead `mapTo(false)` use `mapTo({ type: '[Action type]', pending: true })`
instead `mapTo(true)` use `mapTo({ type: '[Action type]', pending: false })`
 

```ts
actions.pipe(ofActionExecuting(Action1)).subscribe(executing => actionStatus.push(executing));

store.dispatch(new Action1());

expect().toEqual([ { type: 'ACTION 1', pending: true }, {  type: 'ACTION 1', pending: false } ]);

```

But

```ts
actions.pipe(ofActionExecuting([ Action1, Action2 ])).subscribe(executing => actionStatus.push(executing));

store.dispatch([ new Action1(), new Action2() ]);

expect().toEqual([ 
  { type: 'ACTION 1', pending: true }, 
  { type: 'ACTION 2', pending: true }, 

  {  type: 'ACTION 1', pending: false },
  { type: 'ACTION 2', pending: false }, 
]);
```

Multiple

Reply:
`pending` is not a suitable name. Why? `pending` is a synonym to `being soon to appear or take place`. In the context of lyrics if pending is falsy then it means that action appeared (but not handled or anything else, as actions can be async). IMO `executing` takes place.

Reply:
orginally i though of this to replace the use of `setLoading`, `setLoaded` state-operators, 
from your comments i can see how we can expand the use of it, but following the same initial idea, i would expect that `ofActionExecuting` on multiple actions would return true when all were dispatched and false when all were done

btw i also prefer executing vs pending too, but im fine with what you guys decide